<?xml version="1.0"?>
<launch>

    <node pkg="ezw_ros_controllers" name="drive_controller" type="drive_controller" output="screen">
        <rosparam param="pub_freq_hz">10</rosparam>
        <rosparam param="motors_dict">
            [
                {"motor_1", "/opt/ros/share/ezw-ros-controllers/config/motor_1.ini"},
                {"motor_2", "/opt/ros/share/ezw-ros-controllers/config/motor_2.ini"}
            ]
        </rosparam>
        <rosparam param="watchdog_receive_ms">500</rosparam>
    </node>

</launch>
